python/ray/function_manager.py

```diff
@@ -192,24 +358,28 @@ def f():
             self._worker.redis_client.rpush(
                 b"FunctionTable:" + function_id.id(), self._worker.worker_id)
 
-    def get_execution_info(self, driver_id, function_id):
+    def get_execution_info(self, driver_id, function_descriptor):
         """Get the FunctionExecutionInfo of a remote function.
 
         Args:
             driver_id: ID of the driver that the function belongs to.
-            function_id: ID of the function to get.
+            function_descriptor: The FunctionDescriptor of the function to get.
 
         Returns:
             A FunctionExecutionInfo object.
         """
-        # Wait until the function to be executed has actually been registered
-        # on this worker. We will push warnings to the user if we spend too
-        # long in this loop.
+        function_id = function_descriptor.function_id.id()
+
+        # Wait until the function to be executed has actually been
+        # registered on this worker. We will push warnings to the user if
+        # we spend too long in this loop.
+        # The driver function may not be found in worker lib path. Try to load
```

Comment:
What is the "worker lib path"?

Reply:
I think he means sys.path. this sentence should belong to the next PR which enables loading functions from local disk, and should be removed from this PR.